docs/src/main/sphinx/functions/list.rst

```diff
@@ -379,6 +380,7 @@ S
 
 - :func:`second`
 - :func:`sequence`
+- :ref:`sequence() <sequence_table_function>`
```

Comment:
or remove () here ... and instead do a 

sequence (table function) 

or something like that?

Reply:
remove `()`

Reply:
When `()` is removed, `sequence` TF renders different than `exclude_columns` TF. 
I think we should keep `()`, and additionally add a note like `(table function)` for disambiguation.

So, it would be:
`- :ref:`sequence( ) <sequence_table_function>` (table function)`